_posts/2023-11-24-sessions-at-reinvent.md

```diff
@@ -0,0 +1,65 @@
+
+# 3. OPN202 - Logging with Amazon EKS, Bottlerocket, Fluent Bit, and OpenSearch
+
+Time: Nov. 29, 3:00 PM - 5:00 PM (PST)
+<br>Location: [Venetian, Level 3, Murano 3203](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=OPN202)
+
+I am excited to be running this workshop with Kyle Davis, Senior Developer Advocate for Bottlerocket. We will cover how you can do observability on a shell-less cluster. You will deploy an EKS cluster onto nodes running Bottlerocket, a security hardened container focused operating system. Then we will deploy OpenSearch and Fluent Bit to do observability on our cluster. This is an end to end workshop with no prior experience required so we hope to see you there!
+
+# 4. ANT210 - Improve your search with vector capabilities in OpenSearch Service
+
+Time: Nov. 30, 2:30 PM - 3:30 PM (PST)
+<br>Location: [Caesars Forum, Level 1, Summit 201](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT210)
+
+Did you know that OpenSearch is not just a vector database but 3 vector databases in one? With OpenSearch you can leverage vector stores like FAISS, nmslib, and Lucenes Vector Store. In this session Achal Kumar from Intuit along with Aruna Govindaraju and Jon Handler from AWS will talk through how to get more relevant search results with OpenSearch as a vector store. Additionally, you will get to hear first hand how Intuit is using Amazon OpenSearch Service to get more relevant results.
+
+# 5. COM310 - Processing over 450 million events per day in real time with AWS
+
+Time: Nov. 27, 5:30 PM - 5:50 PM (PST)
+<br>Location: [Venetian, Level 2, Hall B, Expo, Booth #1480, Theater 3](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=COM310)
+
+As an observability solution, OpenSearch scales! Hear from Neylson Crepalde at A3Data how they are doing observability on a whopping 5Tb per month of logs. He will cover the architecture used and many of their learnings along the way.
+
+# 6. ANT301 - What’s new in Amazon OpenSearch Service
+
+Time: Nov. 30, 4:00 PM - 5:00 PM (PST)
+<br>Locations:
+* [Caesars Forum, Level 1, Summit 201](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT301)
+* [Mandalay Bay, Level 2, South, Mandalay Bay Ballroom L, Content Hub, Teal Screen](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT301-SC1)
+* [MGM Grand, Level 1, Boulevard 163, Content Hub, Yellow Screen](https://hub.reinvent.awsevents.com/attendee-portal/catalog/?search=ANT301-SC2)
+
+And saving the best for last; make sure you checkout live what is new in the Amazon OpenSearch Service! This session hosted by Mukul (GM of Search Services at AWS), Carl (Director of OpenSearch at AWS), and Bill Burket (Engineering Leader at EA), covers many of the new things that are going to make your experience building on OpenSearch even better! You are not going to want to miss this session.
```

Comment:
```suggestion
And saving the best for last; make sure you checkout live what is new in the Amazon OpenSearch Service! This session hosted by Mukul (GM of Search Services at AWS), Carl (Director of OpenSearch at AWS), and Bill Burket (Engineering Leader at EA), covers many of the new things that are going to make your experience building on OpenSearch even better! You are not going to want to miss this session.
```

Reply:
@Naarcha-AWS hey, this change looks the same as the original text.